ui_framework/test/common_html_props.js

```diff
@@ -0,0 +1,5 @@
+export const commonHtmlProps = {
```

Comment:
I like your suggestion of `required` but not so sure about `support` part because it makes me think the props have something to do with support.  Switched to just `requiredProps`.